src/modules/mc_pos_control/multicopter_altitude_mode_params.c

```diff
@@ -55,6 +55,34 @@ PARAM_DEFINE_FLOAT(MPC_ACC_UP_MAX, 4.f);
  */
 PARAM_DEFINE_FLOAT(MPC_ACC_DOWN_MAX, 3.f);
 
+/**
+ * Maximum upwards velocity in Altitude/Position mode
+ *
+ * Should be less than MPC_Z_VEL_MAX_UP.
+ *
+ * @unit m/s
+ * @min 0.5
+ * @max 8
+ * @increment 0.1
+ * @decimal 1
+ * @group Multicopter Position Control
+ */
+PARAM_DEFINE_FLOAT(MPC_Z_V_MAN_UP, 1.5f);
+
+/**
+ * Maximum downwards velocity in Altitude/Position mode
+ *
+ * Should be less than MPC_Z_VEL_MAX_DN.
+ *
+ * @unit m/s
+ * @min 0.5
+ * @max 4
+ * @increment 0.1
+ * @decimal 1
+ * @group Multicopter Position Control
+ */
+PARAM_DEFINE_FLOAT(MPC_Z_V_MAN_DN, 1.5f);
```

Comment:
```suggestion
PARAM_DEFINE_FLOAT(MPC_Z_V_MAN_DN, 1.f);
```

Reply:
Since `MPC_Z_VEL_MAX_DN` is 1.5, I'd leave this as 1.5 too, unless you think we should be further decreasing the limit?

https://github.com/PX4/PX4-Autopilot/blob/253208fdd4d3b415ef9ce019d7b5475a3e103a8a/src/modules/mc_pos_control/multicopter_position_control_limits_params.c#L81